rotkehlchen/db/dbhandler.py

```diff
@@ -1361,10 +1365,10 @@ def save_tokens_for_address(
         )
         # Delete previous entries for tokens
         write_cursor.execute(
-            'DELETE FROM accounts_details WHERE account=? AND blockchain=? AND KEY=?',
-            (address, blockchain.serialize(), ACCOUNTS_DETAILS_TOKENS),
+            'DELETE FROM accounts_details WHERE account=? AND KEY IN(?, ?)',
```

Comment:
why did you delete the blockchain key here?

Reply:
Ahhh, my bad, probably this happened when I was switching from develop to bugfixes (since in develop `chain_id` is used)